src/helpers.ts

```diff
@@ -237,6 +237,10 @@ export function rewriteErrorStack(stack: string, indexes: [number, number]) {
 // non Error exceptions can be thrown, it tolerates though. The
 // redundant type Error | any expresses that.
 export function formatError(error: Error | any) {
+  if (error === undefined || error === null) {
```

Comment:
error== null covers both. Or we could just do !error. I don't think users would be throwing false 🤷🏽‍♂️